exonum-java-binding-core/src/main/java/com/exonum/binding/blockchain/Blockchain.java

```diff
@@ -71,17 +75,102 @@ public long getHeight() {
   }
 
   /**
-   * Returns a proof list of transaction hashes committed in the block at the given height.
+   * Returns a proof list of transaction hashes committed in the block at the given height or
+   * an empty list if the block at the given height doesn't exist.
    *
    * @param height block height starting from 0
-   * @return a proof list of transaction hashes committed in the block.
-   *        Or an empty list if the block at the given height doesn't exist
    * @throws IllegalArgumentException if the height is negative
    */
   public ProofListIndexProxy<HashCode> getBlockTransactions(long height) {
     return schema.getBlockTransactions(height);
   }
 
+  /**
+   * Returns a proof list of transaction hashes committed in the block with given id or null if
```

Comment:
It's quite weird. There are 3 overloaded methods and in the same case one returns `null` and others return 'empty list'

Reply:
Thanks, I changed it to `empty list` for now, but we'd need to decide whether we want to use nulls, empty lists or exceptions (either checked or unchecked).